### PR TITLE
Add reducers and actions to redux files

### PR DESCRIPTION
**Pull request by MMGGYY66**
# Add reducers and actions to Redux files.
- [Add reducers and actions to Redux files](https://github.com/MMGGYY66/my-react-bookstore/pull/4/commits/73a98cd6aa39250f755d3af3431b81b4e58dd32f)
- [Update components folders and files to connect to store.js](https://github.com/MMGGYY66/my-react-bookstore/pull/4/commits/d92ae0a4a16bf6fdc53c7bc67b1fa4ca605f932a)
- [Fix linters issues](https://github.com/MMGGYY66/my-react-bookstore/pull/4/commits/6261fcb4c047d0ea529248b78df1a8a635345ddf)

### CODE REVIEW

File: src/components/Categories.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import '../cat.css';
+
+function Categories() {
+  return (
+    <div style={{ display: 'flex', justifyContent: 'center' }}>
+      <button type="button">Check status</button>
+    </div>
+  );
+}
```
Comment (AmaduKamara) [inline src/components/Categories.js]: - [ ] For this component, you're expected to dispatch the checkStatus action when the button is clicked so the method dispatches the action and updates the categories state, and then renders the Under construction text on the screen.
- [ ] Kindly import the useDispatch hook and create a dispatch variable, then also create a function in which you dispatch the action which is again imported then you pass this function into the button when it's clicked. 👍🏽

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️ ♻️ ♻️

Hi @MMGGYY66,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✅ You used Gitflow correctly 👍🏽
✅ Linter checks passed 👍🏽
✅ Connected redux with react and adding and removing book works fine 👍🏽

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/components/Categories.js (hunk 1 above)
Comment (Abenezer-Tilahun) [reply src/components/Categories.js]: Fixed.

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @MMGGYY66  
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Git -Flow.
✔️ Professional README  
✔️ Good Commit Messages
✔️ Good job for Implemented Suggested changes.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**